### DISCUSSION TITLE
blog/crud-with-dynamodb-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#23] [General]
# blog/crud-with-dynamodb-in-aspnet-core/

In this article, we are going to learn about implementing CRUD with DynamoDB in ASP.NET Core Web API. This article is an integral part of the entire "Serverless Applications with AWS" which I have been writing on my blog.

https://codewithmukesh.com/blog/crud-with-dynamodb-in-aspnet-core/

**Comment by linhnvv2** [comment]
in
 "AWS": {
    "Profile": "aws-admin",
    "Region": "ap-south-1"
  }
how we connect with muti account. Is it by AWS CLI.
